doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
+
+    -   :ref:`audit_log.syslog_facility <configuration_reference_audit_syslog-facility>` -- a system logger keyword that tells syslogd where to send the message.
+        The default value is ``local7``.
+
+    -   :ref:`audit_log.syslog_identity <configuration_reference_audit_syslog-identity>` -- a string placed at the beginning of every message.
+        The default value is ``tarantool``.
+
+    These options are interpreted as a message for the `syslogd <https://datatracker.ietf.org/doc/html/rfc5424>`_ program,
+    which runs in the background of any Unix-like platform.
+
+    An example of a Tarantool audit log entry in the syslog:
+
+    ..  code-block:: json
+
+        {
+          "__CURSOR" : "s=81564632436a4de590e80b89b0151148;i=11519;b=def80c1464fe49d1aac8a64895d6614d;m=8c825ebfc;t=5edb27a75f282;x=7eba320f7cc9ae4d",
+          "__REALTIME_TIMESTAMP" : "1668725698065026",
+          "__MONOTONIC_TIMESTAMP" : "37717666812",
+          "_BOOT_ID" : "def80c1464fe49d1aac8a64895d6614d",
+          "_UID" : "1003",
+          "_GID" : "1004",
+          "_COMM" : "tarantool",
+          "_EXE" : "/app/tarantool/dist/tdg-2.6.4.0.x86_64/tarantool",
+          "_CMDLINE" : "tarantool init.lua <running>: core-03",
+          "_CAP_EFFECTIVE" : "0",
+          "_AUDIT_SESSION" : "1",
+          "_AUDIT_LOGINUID" : "1003",
+          "_SYSTEMD_CGROUP" : "/user.slice/user-1003.slice/user@1003.service/app.slice/app@core-03.service",
+          "_SYSTEMD_OWNER_UID" : "1003",
+          "_SYSTEMD_UNIT" : "user@1003.service",
+          "_SYSTEMD_USER_UNIT" : "app@core-03.service",
+          "_SYSTEMD_SLICE" : "user-1003.slice",
+          "_SYSTEMD_USER_SLICE" : "app.slice",
+          "_SYSTEMD_INVOCATION_ID" : "be368b4243d842ea8c06b010e0df62c2",
+          "_MACHINE_ID" : "2e2339725deb4bc198c54ff4a2e8d626",
+          "_HOSTNAME" : "vm-0.test.env",
+          "_TRANSPORT" : "syslog",
+          "PRIORITY" : "6",
+          "SYSLOG_FACILITY" : "23",
+          "SYSLOG_IDENTIFIER" : "tarantool",
+          "SYSLOG_PID" : "101562",
+          "_PID" : "101562",
+          "MESSAGE" : "remote: session_type:background module:common.admin.auth user: type:custom_tdg_audit tag:tdg_severity_INFO description:[119eae0e-a691-42cc-9b4c-f14c499e6726] subj: \"anonymous\", msg: \"Access granted to anonymous user\"",
+          "_SOURCE_REALTIME_TIMESTAMP" : "1668725698064202"
+        }
+
+    ..  warning::
+
+        Above is an example of writing audit logs to a directory shared with the system logs.
+        Tarantool allows this option, but it is not recommended to do this to avoid difficulties
+        when working with audit logs. System and audit logs should be written separately.
+        To do this, create separate paths and specify them.
+
+    |
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_AUDIT_LOG_SYSLOG_SERVER
+
+..  _configuration_reference_audit_to:
+
+..  confval:: audit_log.to
```

Comment:
I guess that it is the most commonly used option, but it is the last alphabetically.

It is expected that a reference contains the options in the alphabetical order. Maybe we can highlight this option somehow differently?

I've no certain proposal, just thinking around.

Reply:
The section starts with a link to the 'audit module' article, which is more like a guide. And it describes `audit_log.to` first. It seems, my concern is already solved.